src/components/pages/Contacts/index.tsx

```diff
@@ -55,16 +54,16 @@ const UnconnectedContacts: React.FC<PropsFromRedux> = ({
   );
   const [hasFetchedData, setHasFetchedData] = useState<boolean>(false);
 
+  const contacts = orgContacts.contacts;
   const token = user.user.token;
   const org = user.user.org;
```

Comment:
nit: should we just destruct to be more ES6-y?
const { contacts}  = orgContacts;
const {token} = user.user;
const {org} = user.user

Reply:
that is new syntax to me - but i'm a big fan. Just switched it